fe/fe-core/src/main/java/org/apache/doris/system/Backend.java

```diff
@@ -95,6 +95,8 @@
     // additional backendStatus information for BE, display in JSON format
     private BackendStatus backendStatus = new BackendStatus();
 
+    private String lastStreamLoadTime = "";
```

Comment:
Why not using timestamp as Long?
And this field is not persisted. So every time the FE restart, this field will be reset to
`1970`?

Reply:
> Why not using timestamp as Long?
> And this field is not persisted. So every time the FE restart, this field will be reset to
> `1970`?

I have modified the code and this field will be persisted in bdbje.